test-e2e/src/lib.rs

```diff
@@ -1,20 +1,50 @@
-use mopro_ffi::{app, WtnsFn};
+extern crate core;
 
-rust_witness::witness!(multiplier2);
-rust_witness::witness!(multiplier2bls);
-rust_witness::witness!(keccak256256test);
-rust_witness::witness!(hashbenchbls);
+// First, configure the Mopro FFI library
+mopro_ffi::app!();
 
-app!();
+/// Macro containing the Circom example
+#[cfg(feature = "circom")]
```

Comment:
I don't think we should do conditional compilation in this crate. test-e2e should test all proving systems.

in the test apps we should eventually add more screens to handle generating different types of proofs

Reply:
Removing conditional compilation should make this easier to understand. Macros are scary

Reply:
Cleaned it up, agree that this is a better way, considering test-e2e should test all proving systems.